client/devdocs/design/blocks.jsx

```diff
@@ -62,7 +62,7 @@ import ReaderAvatar from 'blocks/reader-avatar/docs/example';
 import ImageEditor from 'blocks/image-editor/docs/example';
 import VideoEditor from 'blocks/video-editor/docs/example';
 import ReaderPostCard from 'blocks/reader-post-card/docs/example';
-import ReaderCombinedCard from 'blocks/reader-combined-card/docs/example';
+//import ReaderCombinedCard from 'blocks/reader-combined-card/docs/example';
```

Comment:
Must be re-added before 🚢 (after this example is fixed in https://github.com/Automattic/wp-calypso/pull/23166).